src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,35 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new IllegalArgumentException("User cannot be null");
```

Comment:
![image](https://github.com/mate-academy/jv-registration-validation/assets/20326858/9e0db751-c608-4dc6-af92-49ec42269d88)

Reply:
not fixed